e2e/tests/functional/couchdb.e2e.spec.js

```diff
@@ -95,14 +95,10 @@ test.describe("CouchDB initialization @couchdb", () => {
         // Go to baseURL
         await page.goto('./', { waitUntil: 'networkidle' });
 
-        // Verify that error banner is displayed
-        const bannerMessage = await page.locator('.c-message-banner__message').innerText();
-        expect(bannerMessage).toEqual('Failed to retrieve object mine');
-
         // Verify that a PUT request to create "My Items" folder was made
         await expect.poll(() => createMineFolderRequests.length, {
             message: 'Verify that PUT request to create "mine" folder was made',
-            timeout: 1000
```

Comment:
Why the time change?

Reply:
I think since I removed the wait for banner message (it doesn't pop up anymore), one second is now too short of a time to poll (test was flaking due to missed request).

I might instead await a promise so we don't need to poll